.sync.yml

```diff
@@ -7,6 +7,37 @@
       - GOOS: darwin
         GOARCH: amd64
 .travis.yml:
+  sudo: true
+  services:
+    - docker
+  env:
+    global:
+      - ORG_PATH=/home/travis/gopath/src/github.com/intelsdi-x
+      - SNAP_PLUGIN_SOURCE=/home/travis/gopath/src/github.com/${TRAVIS_REPO_SLUG}
+    matrix:
+      - TEST_TYPE: small
+      - SNAP_VERSION: latest
+        SNAP_OS: alpine
+        TEST_TYPE: large
+      - SNAP_VERSION: latest_build
```

Comment:
So not sure when we will release a new version of snap. For now only test on latest_build. When PR 1349 is merged, only latest_build will pass.